packages/sanity/src/form/studio/inputResolver/resolveArrayInput.ts

```diff
@@ -30,15 +30,15 @@ export function hasOptionsList(type: ArraySchemaType): boolean {
 }
 
 export function resolveArrayInput(type: ArraySchemaType): ComponentType<any> {
+  if (isStringArray(type) && isTagsArray(type)) {
```

Comment:
just out of curiosity: was moving this to the top here required?

Reply:
I moved it because the tag layout didn't render if you also had `list` in `options`. Not sure what is the right thing to do here. I'm going to move it back to how it was before for now 👍